adventurelog/umbrel-app.yml

```
@@ -0,0 +1,25 @@
+manifestVersion: 1
+id: adventurelog
+category: files
+name: AdventureLog
+icon: https://raw.githubusercontent.com/seanmorley15/AdventureLog/refs/heads/main/brand/adventurelog.png
+version: "0.8.0"
+tagline: adventurelog
+description: adventurelog
```

Add a proper tagline and description.
```suggestion
tagline: Self-hostable travel tracker and trip planner.
description: >-
  🗺️ Starting from a simple idea of tracking travel locations (called adventures), AdventureLog has grown into a full-fledged travel companion. With AdventureLog, you can log your adventures, keep track of where you've been on the world map, plan your next trip collaboratively, and share your experiences with friends and family.


  AdventureLog was created to solve a problem: the lack of a modern, open-source, user-friendly travel companion. Many existing travel apps are either too complex, too expensive, or too closed-off to be useful for the average traveler. AdventureLog aims to be the opposite: simple, beautiful, and open to everyone.
```